README.md

```diff
@@ -67,6 +71,8 @@ Connecting to log stream...
 
 * Local Git deployment may fail with large commits
 
+<!-- region exclude-from-marketplace -->
```

Comment:
How come we want to remove this from the marketplace btw?

Reply:
Just didn't feel like this info was relevant for people viewing our marketplace page. The typical user that installs our extension doesn't care about how to debug our extension or how to sign a CLA. And for the few users that actually want to contribute, there's a "Repository" button to take them to the source code where they can see this info

<img width="715" alt="Screen Shot 2020-08-26 at 1 44 17 PM" src="https://user-images.githubusercontent.com/11282622/91355031-714fe100-e7a2-11ea-8c56-2a6687bafde7.png">